app/src/main/java/com/example/sarahtang/mdbsocials/SocialsAdapter.java

```diff
@@ -0,0 +1,161 @@
+package com.example.sarahtang.mdbsocials;
+
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.net.Uri;
+import android.os.AsyncTask;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.CardView;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.storage.FirebaseStorage;
+
+import java.util.ArrayList;
+
+import static android.content.Intent.FLAG_ACTIVITY_NEW_TASK;
+
+/**
+ * Created by sarahtang on 2/21/17.
+ * Connects social layout with social information.
+ */
+
+public class SocialsAdapter extends RecyclerView.Adapter<SocialsAdapter.CustomViewHolder> {
+    private Context context;
+    public View view;
+    ArrayList<Social> listSocials;
+
+    public static final String NAME_KEY = "name";
+    public static final String HOST_KEY = "host";
+    public static final String DESCRIPTION_KEY = "description";
+    public static final String PEOPLE_INTERESTED = "peopleInterested";
+    public static final String NUMBER_INTERESTED = "numberInterested";
+    public static final String FIREBASE_URL = "firebaseURL";
+    public static final String FIREBASE_KEY = "firebaseKey";
```

Comment:
you use these keys in other classes too, better to just put them in a Utils class